pkg/cmd/roachtest/roachtestutil/mixedversion/runner.go

```diff
@@ -458,7 +458,7 @@ func (tr *testRunner) maybeInitConnections() error {
 
 	cc := make([]*gosql.DB, len(tr.crdbNodes))
 	for _, node := range tr.crdbNodes {
-		conn, err := tr.cluster.ConnE(tr.ctx, tr.logger, node)
+		conn, err := tr.cluster.ConnE(tr.ctx, tr.logger, node, option.DBName("system"))
```

Comment:
If we're doing this, then we don't need to change `mixed_version_backup.go`, right?

Reply:
yeah you are right. I added the change in `mixed_version_backup` so we didn't have to change the connection DB for everyone. I removed this.